packages/protocol/thread/pause.ts

```diff
@@ -242,11 +248,20 @@ export class Pause {
       if (!this.frames.has(f.frameId)) {
         this.frames.set(f.frameId, f as WiredFrame);
       }
+      if (!this.rawFrames.has(f.frameId)) {
+        const rawFrame = cloneDeep(f);
+        this.ThreadFront.updateMappedLocation(rawFrame.location);
+        this.ThreadFront.updateMappedLocation(rawFrame.functionLocation);
```

Comment:
:speech_balloon: and just filed FE-756 to cover removing the need for `updateMappedLocation` entirely